src/engine/resolvers.rs

```diff
@@ -80,11 +82,60 @@ where
         }
     }
 
+    /// Returns a neo4j database driver from the pool.
+    ///
+    /// # Errors
+    ///
+    /// Returns an [`Error`] variant [`DatabaseMismatch`] if the feature `neo4j`
+    /// is not enabled.
+    ///
+    /// Returns an [`Error]` variant [`Neo4jPoolGetConnectionFailed`] if a new
+    /// connection to the database cannot be retrieved from the pool.
+    ///
+    /// [`Error`]: ../../error/enum.Error.html
+    /// [`DatabaseMismatch`]: ../../error/enum.Error.html#variant.DatabaseMismatch
+    /// [`Neo4jPoolGetConnectionFailed`]: ../../error/enum.Error.html#variant.Neo4jPoolGetConnectionFailed
+    ///
+    /// # Examples
+    ///
+    /// ```rust, no_run
+    /// # use tokio::runtime::Runtime;
+    /// # use warpgrapher::engine::resolvers::{ResolverFacade, ExecutionResult};
+    ///
+    /// fn custom_resolve(facade: ResolverFacade<(), ()>) -> ExecutionResult {
+    ///     let mut rt = Runtime::new().unwrap();
+    ///     rt.block_on(async {
+    ///
+    ///         let neo4j_client = facade.db_into_neo4j().await.unwrap();
+    ///         // use neo4j client
+    ///
+    ///     });
+    ///     facade.resolve_null()
+    /// }
+    /// ```
+    #[cfg(feature = "neo4j")]
+    pub async fn db_into_neo4j(
```

Comment:
Unless overruled, I still think it makes more sense for this into operation to be on the pool itself, so that the call chain for a client would look like `facade.pool().into_neo4j() ` rather than `facade.db_into_neo4j()`.  Or given that this is a facade that's supposed to make life easier, I think I'd be fine with a `facade.db_into_neo4j()` call, as the interface is here, but with the behavior delegated under the hood so that the code in this function is just a one-line `self.executor().pool().into_neo4j()`.  The functionality for doing the conversion into a Neo4J client should live on the pool, though, not in the facade, I think.

Reply:
Fixed.